content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+# data status
+
+Show changes in data tracked by DVC between the last commit, the dvcfiles and
+the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
```

Comment:
reminder: need to update syntax highlighters for this

Reply:
Took a long time for me to find out how to do this. :)

Reply:
is it solved?

Reply:
yeah, it should be.